client/my-sites/plugins/education-footer/index.tsx

```diff
@@ -78,14 +79,19 @@ export const MarketplaceFooter = () => {
 	const { __ } = useI18n();
 	const isLoggedIn = useSelector( isUserLoggedIn );
 
+	const sectionName = useSelector( getSectionName );
+
+	let startUrl = sectionName === 'plugins' ? '/start/business' : '/start';
+	startUrl += `/?ref=${ sectionName }-lp`;
```

Comment:
Looks like we can use `{ addQueryArgs } from 'calypso/lib/route'` here.

Reply:
Thanks for the tip!